bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh

```diff
@@ -234,14 +234,25 @@ pgbouncer_initialize() {
         ini-file set --ignore-inline-comments --section "databases" --key "$PGBOUNCER_DATABASE" --value "$database_value" "$PGBOUNCER_CONF_FILE"
 
         i=0;
-        while true; VAR_NAME="PGBOUNCER_DSN_${i}";
+        while true; VAR_NAME="PGBOUNCER_DSN_${i}"; FILE_VAR_NAME="PGBOUNCER_DSN_${i}_FILE";
         do
-            if [ -z "${!VAR_NAME+x}" ]; then
-                break;
-            else
+            if [ -n "${!FILE_VAR_NAME+x}" ]; then
+                debug "reading \$$VAR_NAME from file, via \$$FILE_VAR_NAME (${!FILE_VAR_NAME})"
+                if [[ -r "${!FILE_VAR_NAME:-}" ]]; then
+                    export "${VAR_NAME}=$(< "${!FILE_VAR_NAME}")"
+                    unset "${FILE_VAR_NAME}"
+                else
+                    error "Failed to read file specified in \$$FILE_VAR_NAME. '${!FILE_VAR_NAME:-}' is not readable."
+                    exit 1
```

Comment:
Although we see the point in stopping the initialization process here in case one of the specified files is not readable, we would like to be consistent with our current behavior for this situation (see [pgbouncer-env.sh file](https://github.com/bitnami/containers/blob/5d2cd9c5ca1a82fac367c986d6097521fcb914cc/bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh#L98)), and default to skip. 

Can you also add a new environment variable to control the skip or fail outcome?

Reply:
Also please remember to document the new env vars in the `README.md` file.